packages/yourloops/components/dialogs/weekly-report-dialog.tsx

```diff
@@ -0,0 +1,169 @@
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { makeStyles, Theme } from "@material-ui/core/styles";
+
+import DescriptionOutlinedIcon from "@material-ui/icons/DescriptionOutlined";
+
+import Box from "@material-ui/core/Box";
+import Button from "@material-ui/core/Button";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import Typography from "@material-ui/core/Typography";
+
+import { WeeklyReport } from "../../lib/medical-files/model";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import Divider from "@material-ui/core/Divider";
+import { useTeam } from "../../lib/team";
+
+interface Props {
+  onClose: () => void;
+  weeklyReport: WeeklyReport;
+}
+
+const classes = makeStyles((theme: Theme) => ({
+  divider: {
+    margin: "30px 0 10px 16px",
+  },
+  title: {
+    "display": "flex",
+    "alignItems": "center",
+    "justifyContent": "center",
+    "& > svg": {
+      marginRight: theme.spacing(1),
+    },
+  },
+  explanatoryText: {
+    marginTop: theme.spacing(3),
+    fontStyle: "italic",
+  },
+}));
+
+export default function WeeklyReportDialog(props: Props): JSX.Element {
+  const { title, divider, explanatoryText } = classes();
+  const { t } = useTranslation("yourloops");
+  const { onClose, weeklyReport } = props;
+  const teamHook = useTeam();
+  const patient = teamHook.getPatient(weeklyReport.patientId);
+  const endDatePeriod = new Date(weeklyReport.creationDate);
+  const startDatePeriod = new Date(weeklyReport.creationDate);
+  startDatePeriod.setDate(startDatePeriod.getDate() - 7);
+
+  return (
+    <Dialog
+      open
+      fullWidth
+      maxWidth="md"
+      onClose={onClose}
+    >
+      <DialogTitle>
+        <Box className={title}>
+          <DescriptionOutlinedIcon />
+          <Typography variant="h5">
+            {t("weekly-report-pdf", { pdfName: new Date(weeklyReport.creationDate).toLocaleDateString() })}
```

Comment:
toLocaleDateString is not adequate depending on the current lang. This can be fixed by using moment.js.

Something like that should do the trick:
`moment.utc(weeklyReport.creationDate).format(moment.localeData().longDateFormat("ll")).toString()`

(example found at packages/yourloops/components/patient/row.tsx line 118)

Reply:
import moment from "moment-timezone";